ksp_plugin_adapter/flight_planner.cs

```diff
@@ -139,6 +139,14 @@ class FlightPlanner : VesselSupervisedWindowRenderer {
 
   private void RenderFlightPlan(string vessel_guid) {
     using (new UnityEngine.GUILayout.VerticalScope()) {
+      if (UnityEngine.GUILayout.Button("Rebase")) {
```

Comment:
It seems odd to put this button at the very top of the window.  It doesn't affect the parameters that are immediately below (steps, tolerance).  I would put it closer to the manœuvres that it affects, i.e., near line 232.